STYLE.md

```diff
@@ -19,97 +19,138 @@ We follow the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-gu
+  - **BAD**: It's _very important_ that you click the stop button after running **./script.sh**.
+  - **BETTER**: Click the **Stop** button after running `./script.sh`.
 
-  **Bad**: Now that we're happy with the terrain, we can start placing our models.
+- Avoid idioms or colloquialisms. They're hard to localize and confuse many readers.
 
-  **Better**: When you're satisfied with the terrain, begin placing your models.
+  - **WORSE**: Publishing in Roblox is a piece of cake.
+  - **BETTER**: Publishing in Roblox is fast and simple.
 
-- Use bold for key terms and UI elements. Use monospace for file paths, code, functions, variables, etc. Don't use italics.
+- "Can" is for optional actions or permission, "might" for possibility, "must" for requirements, and "should" for recommendations. Avoid "may."
 
-  **Bad**: It's _very important_ that you click the stop button after running ./script.sh.
+  - **WORSE**: You should accept the license agreement before you may make changes. You might also review the coding standards.
+  - **BETTER**: You must accept the license agreement before you can make changes. You should also review the coding standards.
 
-  **Better**: Click the **Stop** button after running `./script.sh`.
+- Avoid gender identifiers. Whenever possible, make subjects plural.
 
-- Avoid idioms or colloquialisms. They're hard to localize and confuse many readers.
+  - **BAD**: After clicking, a player can see his or her inventory on his/her screen.
+  - **BETTER**: After clicking, players can see their inventories on screen.
 
-  **Bad**: Publishing in Roblox is a piece of cake.
+- Use "select" for options or objects in an experience. Use "click" for buttons or icons.
 
-  **Better**: Publishing in Roblox is fast and simple.
+  - **BAD**: Click and drag to grab the signpost. Then select the **Color** button.
+  - **BETTER**: Select the signpost and click the **Color** button.
 
-- "Can" is for optional actions or permission, "might" for possibility, "must" for requirements, and "should" for recommendations. Avoid "may."
+- Spell out one (1) through nine (9). Start using numerals at 10. If a number needs a unit (GB, pounds, millimeters, kg, celsius, etc.), always use numerals, even if the number if smaller than 10.
 
-  **Bad**: You should accept the license agreement before you may make changes. You might also review the coding standards.
+  - **BAD**: 3 people looked for thirteen files on a six GB hard drive.
+  - **BETTER**: Three people looked for 13 files on a 6 GB hard drive.
 
-  **Better**: You must accept the license agreement before you can make changes. You should also review the coding standards.
+## Links
 
-- Avoid gender identifiers. Whenever possible, make subjects plural.
+### Off-Site Links
 
-  **Bad**: After clicking, a player can see his or her inventory on his/her screen.
+For links to other domains or the web, use standard Markdown instead of JSX or HTML:
```

Comment:
```suggestion
For links to other domains or the web outside of [create.roblox.com/docs](https://create.roblox.com/docs), use standard Markdown instead of JSX or HTML:
```